### DISCUSSION TITLE
Count Gold income per map.

### DISCUSSION THREAD

**Discussion by BobMahoo** [Lailloken/Exile-UI#453] [General]
Is there any way to count (or see) the Gold income per map?
Maybe it can be added. :)

thx

**Comment by Lailloken** [comment]
That would only work via screen-reading, and tests in the past have shown that numbers in PoE are hard to read correctly. The gold-text in the inventory is also really small, which makes it even harder.

**Comment by dmitryarch** [comment]
Since gold is a core mechanic now i would like to see this feature as well. Would be useful in map tracker to test various gold-oriented farming strategies. Maybe some workaround could be found. Or maybe something already found cause its almost a year from the last reply. Maybe input gold count manually before map and after and tool will calculate the difference and log it with other map info.

**Reply by Lailloken** [reply]
If you're testing specific strats, I guess you could use the [custom-notes](https://github.com/Lailloken/Exile-UI/wiki/Map%E2%80%90Tracker#logging-custom-notes-for-each-run) feature in the map-tracker. This also allows you to take very specific notes for the strat, as well as easily including map-device items you used (scarabs, allflames, etc.). The way I would do it is:
- dump all gold into the Kingsmarch treasury

- run map

- save the gold amount after the map as a note, like this:

  ![image](https://github.com/user-attachments/assets/086b8f37-90ad-4a6c-8be3-6ae484fc8a5c)

- then repeat

The notes will be stored like other stats after entering a new map. Using the ` x# ` tag means that a note is only used for x runs and then automatically removed:
![image](https://github.com/user-attachments/assets/f3b33b59-e67b-4947-8702-1b377920e82c)

![image](https://github.com/user-attachments/assets/0771044a-6ff4-4f8b-925d-6dda76c40d09)
